assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/NoPricesBanner.tsx

```diff
@@ -4,11 +4,20 @@ import { __ } from '@wordpress/i18n';
 import { Banner } from '@application/ui/display';
 
 import AddDefaultPricesButton from '../buttons/AddDefaultPricesButton';
+import { TicketPriceCalculatorProps } from './TicketPriceCalculator';
 
-const NoPricesBanner = () => {
-	const description = __(
-		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed. Click the save button below to assign which dates this ticket will be available for purchase on.'
-	);
+interface Props extends Pick<TicketPriceCalculatorProps, 'isWithinStepper'> {}
+
+const NoPricesBanner: React.FC<Props> = ({ isWithinStepper }) => {
+	let descriptionText = 'Click the button below to load your default prices into the calculator.';
+
+	if (isWithinStepper) {
+		descriptionText =
+			descriptionText +
+			' Additional ticket price modifiers can be added or removed. Click the save button below to assign which dates this ticket will be available for purchase on.';
+	}
+
+	const description = __(descriptionText);
```

Comment:
?!?!?! oops...

can't run variables through i18n functions!!!

maybe just change that var name to `description` if that's what you need to pass to the component

Reply:
sorry for my ignorance, updated

Reply:
no worries dude... I've done the same thing before too